google-home-community.groovy

```diff
@@ -1311,7 +1317,9 @@ private handleExecuteRequest(request) {
             }
         }
         // Wait up to 5 seconds for devices to report their new state
-        for (def i = 0; i < 50; ++i) {
+        // Reduce wait time to 0.1 second if immediate response requested
+        def waitTime = (settings.immediateResponse) ? 1 : 50
+        for (def i = 0; i < waitTime; ++i) {
```

Comment:
I think it would make more sense to skip over this loop entirely; either by setting `waitTime` to `0` or wrapping the whole loop in an `if (!settings.immediateResponse)`.

Reply:
Agreed